publish-github-release/action.yml

```diff
@@ -20,82 +12,20 @@ inputs:
     required: false
     default: 'dist'
     type: string
-  cleanup_dependencies:
-    required: false
-    type: boolean
-    default: false
   UNIQUES_GITHUB_ACCESS_TOKEN:
     required: true
     type: string
-  NODE_AUTH_TOKEN:
-    required: true
-    type: string
 
 runs:
   using: "composite"
   steps:
-    - uses: actions/setup-node@v3
-      with:
-        node-version: ${{ inputs.node_version }}
-
-    - name: Checkout application repository
-      uses: actions/checkout@v3
-
-    - name: Get NPM cache directory
-      id: npm-cache-dir
-      run: echo "dir=$(npm config get cache)" >> ${GITHUB_OUTPUT}
-      shell: bash
-
-    - name: Cache NPM dependencies
-      id: cache-npm
-      uses: actions/cache@v3
-      with:
-        path: ${{ steps.npm-cache-dir.outputs.dir }}
-        key: ${{ runner.os }}-node-${{ hashFiles('package.json,package-lock.json') }}
-        restore-keys: |
-          ${{ runner.os }}-node-
-
-    - name: Prepare .npmrc file
-      run: |
-        if [ ! -f ".npmrc" ]; then
-          echo "@uniquesca:registry=https://npm.pkg.github.com" >> .npmrc
-          echo "//npm.pkg.github.com/:_authToken=${{ inputs.UNIQUES_GITHUB_ACCESS_TOKEN }}" >> .npmrc
-          echo "always_auth=true" >> .npmrc
-        fi
-      shell: bash
-
-    - name: Install NPM dependencies
-      env:
-        NODE_AUTH_TOKEN: ${{ inputs.NODE_AUTH_TOKEN }}
-      run: npm install --no-dev
-      shell: bash
-
-    - name: Run build command if specified
-      run: ${{ inputs.build_cmd }}
-      shell: bash
-
-    - name: Remove dependencies from package.json
-      if: ${{ inputs.cleanup_dependencies == 'true' }}
-      run: |
-        npm pkg delete dependencies
-        npm pkg delete devDependencies
-        npm pkg delete scripts
-      shell: bash
-
-    #      - name: Publish NPM package on GitHub
-    #        env:
-    #          NODE_AUTH_TOKEN: ${{ inputs.NODE_AUTH_TOKEN }}
-    #        run: |
-    #          npm publish
-    #        shell: bash
-
     - uses: montudor/action-zip@v1
-      if: ${{ inputs.github_release == 'true' }}
+      if: ${{ inputs.github_release == true }}
```

Comment:
Need to remove this if. This action will always do github release, it is it's purpose